pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go

```diff
@@ -0,0 +1,278 @@
+package jobruntestcaseanalyzer
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+
+	"k8s.io/apimachinery/pkg/util/clock"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+	"github.com/openshift/ci-tools/pkg/junit"
+)
+
+const (
+	installTest = "cluster install should succeed"
+)
+
+type requiredPasses struct {
+	testName               string
+	requiredNumberOfPasses int
```

Comment:
make this a pass/fail interface that takes the number of successes and the number of failures